wave/src/main/resources/org/waveprotocol/box/webclient/search/Search.css

```
@@ -65,3 +65,129 @@ input.query:focus {
+  font-size: 11px;
+  text-transform: uppercase;
+  letter-spacing: 0.03em;
+  border-bottom: 2px solid #b0c4d8;
+  padding-bottom: 4px;
+}
+
+.helpTable code {
+  background: #e6f0fa;
+  padding: 1px 4px;
+  border-radius: 3px;
+  font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, monospace;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)